lib/active_model/serializer/adapter.rb

```diff
@@ -22,7 +22,10 @@ def serializable_hash(options = nil)
 
       def as_json(options = nil)
         hash = serializable_hash(options)
-        include_meta(hash) unless self.class == FlattenJson
+        unless self.class == FlattenJson
+          include_meta(hash)
+          include_links(hash)
+        end
```

Comment:
This means we are also adding links to the usual `JsonAdapter`. Any particular reasons for that?